pages/docs/index.tsx

```diff
@@ -22,9 +23,10 @@ const verifySetup = [
 	"Inside the pull request, where you can see the file changes, you will see the parts that are relevant for you highlighted in yellow."
 ]
 
-const Docs = ({ bitbucket_auth_url }: { bitbucket_auth_url: string }) => {
+const Docs = ({ bitbucket_auth_url }: { bitbucket_auth_url: string }, { image_name }: { image_name : string }) => {
 	const { rudderEventMethods } = React.useContext(RudderContext);
 	const session: Session | null = useSession().data;
+	console.log("[Docs] session = ", session)
```

Comment:
Please remove logs after testing. If you want them to stay, convert them to `console.debug` logs.